recipes/AISHELL-1/ASR/seq2seq/hparams/train.yaml

```
@@ -203,22 +203,33 @@ pretrainer: !new:speechbrain.utils.parameter_transfer.Pretrainer
    paths:
       tokenizer: !ref <tokenizer_file>
 
+# Scorer
+ctc_scorer: !new:speechbrain.decoders.scorer.CTCScorer
+   eos_index: !ref <eos_index>
+   blank_index: !ref <blank_index>
+   ctc_fc: !ref <ctc_lin>
+
+scorer: !new:speechbrain.decoders.scorer.ScorerBuilder
+   full_scorers: [!ref <coverage_scorer>, !ref <ctc_scorer>]
+   weights:
+      coverage: !ref <coverage_penalty>
+      ctc: !ref <ctc_decode_weight>
+
 beam_search: !new:speechbrain.decoders.S2SRNNBeamSearcher
```

these will need to be refactored given they are now in the seq2seq.py

---

Can you specify which part should be refactored? 
We use `__init__.py` to automatically import modules in `seq2seq.py`. Maybe we should add `score.py` as well.

---

I should've been more clear sorry about that:
```suggestion
beam_search: !new:speechbrain.decoders.seq2seq.S2SRNNBeamSearcher
```

---

Hi, as mentioned, please refer to `__init__.py` in decoders. Thanks.